### PR TITLE
Fully Optimize Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by F-fengzi**

### CODE REVIEW

File: lang/zh-CN.lng [hunk 1]
```diff
@@ -121,40 +121,40 @@ ConfigurationTabControl.MainTabPage=常规
...
+LZXCompressionLabel=LZX 安装压缩
+LZXCompression.YES=已启用
+LZXCompression.NO=已禁用
 
 [SetupProgressPage]
 PageTitle=安装
-InstallInfoBox=此过程可能需要很长时间，具体取决于你的SD卡性能。
-TryAgainBtn=重试一遍
-InstallStatusLabel.GettingReady=准备就绪
+InstallInfoBox=此过程可能需要很长时间，具体取决于您的SD卡性能。
```
Comment (mariobalanica) [inline lang/zh-CN.lng:151]: Can you replace "SD" with "device" like in the English translation? https://github.com/worproject/WoR-Translations/blob/dc56b2b88ff664f18dfc9c5d1dfb027750c65071/lang/en-US.lng#L151

**Review by mariobalanica — CHANGES_REQUESTED** [review]

File: lang/zh-CN.lng (hunk 1 above)
Comment (F-fengzi) [reply lang/zh-CN.lng:151]: sure!

**Review by mariobalanica — APPROVED** [review]

**Comment by mariobalanica** [comment]
Thanks!

**Comment by F-fengzi** [comment]
Np!

**Pull request merged**